src/Components/ArticleById.jsx

```diff
@@ -55,6 +58,7 @@ export default function ArticleById() {
             }}
           >
             <p>{articles[0].body}</p>
+            <button className="btn btn-outline-primary" onClick={(e)=>{navigate('/Comments',{state:{articles}})} }>Comment</button>
```

Comment:
Passing the state through navigate will only work if your user clicks that button. 

If a user goes straight to `/Comments` (which should be in lowercase btw) that state won't exist.